src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 50px;
+  padding-left: 40px;
+}
+
+.ul {
+  display: flex;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  height: 100%;
+}
+
+img {
+  height: 40px;
+  margin: 10px;
+  margin-top: 6px;
+}
+
+.is-active::after {
+  content: "";
+  color: white;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  border-radius: 8px;
+  height: 4px;
+  display: inline-block;
+  position: absolute;
+  background-color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  margin-right: 20px;
+  height: 60px;
+}
+
+.item_last {
+  margin-right: 0;
+}
+
+a[data-qa="hover"]:hover {
+  color: #00acdc;
+}
+
+.nav__list {
+  padding-bottom: 5px;
+  height: 60px;
+}
+
+a {
+  height: 60px;
+  line-height: 60px;
```

Comment:
try to use paddings top and bottom

Reply:
probably you need to write `display: inline-block;` and then add paddings